syntax = "proto3";

package dependency;

message UserInfo {
  string user_id = 1;
  string email = 2;
  int32 age = 3;
}
